testapi.pm

```diff
@@ -789,15 +789,18 @@ sub is_serial_terminal {
 
 =head2 wait_serial
 
-  wait_serial($regex or ARRAYREF of $regexes [[, $timeout], $expect_not_found]);
+  wait_serial($regex or ARRAYREF of $regexes [, $timeout [, $expect_not_found [, @args ]]]);
+                                                no_regex => 0]);
```

Comment:
you need to delete this line which is dangling around :)

Reply:
Yes.  Done!